code-challenges.test.js

```diff
@@ -13,39 +13,94 @@
+    return arr.map(num => {
+      // each iteration multiplies the current value by 3
+      // the product is returned in the new array made by .map
+      return num * 3
+    })
+  }
+
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+describe('divisibleByThree', () => {
+  it('that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.', () => {
+    const object1 = { number: 15 }
+    // Expected output: "15 is divisible by three"
+    const object2 = { number: 0 }
+    // Expected output: "0 is divisible by three"
+    const object3 = { number: -7 }
+    // Expected output: "-7 is not divisible by three"
+    expect(divisibleByThree(object1)).toEqual("15 is divisible by three")
+    expect(divisibleByThree(object2)).toEqual("0 is divisible by three")
+    expect(divisibleByThree(object3)).toEqual("-7 is not divisible by three")
+  })
+})
+//ReferenceError: divisibleByThree is not defined- good failure
 
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
 
+//input: objects
+//output: divisible by three or not
+const divisibleByThree = (obj) => {
+//use dataype number
+  const number = obj.number
+//use modulo to check number
+    if (number % 3 === 0) {
+        return `${number} is divisible by three`
+    } else {
+        return `${number} is not divisible by three`
+    }
+}
+
+
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
-// Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
 
+describe('capFirstLetter', () => {
+  it('takes in an array of words and returns an array with all the words capitalized', () => {
+    const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+    // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+    const randomNouns2 = ["temperature", "database", "chopsticks", "mango"]
+    // Expected output: ["Temperature", "Database", "Chopsticks", "Mango"]
+    expect(capFirstLetter()).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+    expect(capFirstLetter()).toEqual(["Temperature", "Database", "Chopsticks", "Mango"])
```

Comment:
There should be arguments in each of these function calls

Reply:
i didnt get to finish this one , do i finish it and re push everything?